lib/aws_extensions/ec2/Instance.rb

```diff
@@ -0,0 +1,29 @@
+module AwsExtensions
+  module EC2
+    module Instance
+
+      # Public: Returns the value of the "Name" tag for the Instance
+      def name
+        self.tags.select { |tag| tag.key == "Name" }.first.value
+      rescue
+      	nil
+      end
+
+      # Public: Returns an rray of the block device mappings that are not for the root device
```

Comment:
missing an a here. Either put the a in or use an apostrophe, as in 'rray, the cowboy coder's version of array.